### ISSUE TITLE
RandomFlip3D doesn't work for segmentation task

### ISSUE THREAD

**Issue by filaPro** [open-mmlab/mmdetection3d#1169]
Looks like this [empty_box](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/pipelines/transforms_3d.py#L117) hack is not working for segmentation task as `box_type_3d` is not defined. So, probably here we want to flip point not throw `BaseInstance3DBoxes` but throw `BasePoints`:
```python
if 'box_type_3d' not in input_dict:
    input_dict['points'].flip(direction)
    return
```

**Comment by ZCMax** [comment]
It seems that RandomFlip3D doesn't work for segmentation task indeed, thanks for offering a good solution.

**Comment by Tai-Wang** [comment]
Exactly. We only consider the 3D detection case when adopting that hack to achieve flip augmentation. We will take your kind advice and fix that problem ASAP.

**Comment by ZCMax** [comment]
Fix in #1181
